docs/plans/budget-objectifs.md

```
@@ -0,0 +1,1110 @@
+> - Si `docs/plans/` apparaît dans un `git status` avant commit, **ne pas l'inclure**
+> - Ne pas non plus l'ajouter à `.gitignore` : ce fichier est suivi et partagé avec
+>   l'amont, l'y écrire ferait fuiter l'existence de ces notes
+
+---
+
+## Vue d'ensemble
+
+Deux chantiers liés par une dépendance unique — le lot A3 (trésorerie dans le budget) a besoin que
+les objectifs déclarent correctement ce qu'ils réservent.
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)